docs/general/standalone-support.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Standalone Support
+description: Describes how games can receive standalone support, as well as the conditions- and requirements.
```

Comment:
```suggestion
description: Guide to RetroAchievements support for standalone games, including requirements, evaluation criteria, and the proposal process for games that don't use emulators.
```

The `description` field is what immediately appears when someone finds the page using a search engine. This content here:

![Screenshot 2025-05-05 at 7 50 13 PM](https://github.com/user-attachments/assets/ef1b0085-b3ab-4be3-9052-12aa7dee8358)

How do you feel about this suggestion? It might help the page make more sense if someone is about to land on it sideways from Google (I suspect that's how this page will most often be found).